### DISCUSSION TITLE
designable中属性配置的字段标识可以自定义吗？

### DISCUSSION THREAD

**Discussion by manong333** [alibaba/formily#2293] [Q&A 1.x]
![image](https://user-images.githubusercontent.com/76724683/137055625-83aaf482-b7ba-49cc-bec7-86fd388e76df.png)

**Comment by robin536180** [comment]
可以

**Reply by ThorSmallice** [reply]
如何配置这里的字段属性 是否必填、默认值、是否禁用

**Reply by xkongstar** [reply]
有解决嘛

**Reply by robin536180** [reply]
SettingsForm 里面监听filed字段标识或者组件名称 来配置当前field的状态，或者 配置字段的配置项的时候 重新定义createFieldSchema方法
![77d3925b1564693ae2b755d538db071](https://github.com/user-attachments/assets/36420230-90aa-4ae9-ae3f-f9687a9b353d)
